pkg/docker/util.js

```diff
@@ -595,6 +595,72 @@
         return deferred.promise();
     };
 
+    util.delete_image_confirm = function confirm(client, image) {
+        var deferred = $.Deferred();
+
+        $('#delete-image-confirmation-dialog-title').text(
+            cockpit.format(_("Please confirm deletion of $0"), image.name));
+
+        $('#delete-image-confirmation-dialog-containers>table>tbody').empty();
+        $('#delete-image-confirmation-dialog-body div:nth-child(2)').empty();
+        client.containers_for_image(image.image_id).then(function(containers) {
+            var running_containers = false;
+            $('#delete-image-confirmation-dialog-body div:first-child()').text(
```

Comment:
I agree this text tries to address a bit to many maybes. Would you mind keeping the old text?
We can address the text in another PR if it's a major issue, but I would split it out for now.